_docs/_latest/configure/scanner/rules.md

```diff
@@ -339,21 +469,41 @@ rules:
       - 'storage-api.googleapis.com'
       - 'storage-component.googleapis.com'
  ```
-
-- **name**: The description of your rule.
-- **mode**: A string of one of the following values:
-  - **whitelist**: Allow only the APIs listed in `services`.
-  - **blacklist**: Block the APIs listed in `services`.
-  - **required**: All APIs listed in `services` must be enabled.
-- **resource**: The resource under which the projects reside.
-- **type**: A string of one of the following values:
-  - **organization**
-  - **folder**
-  - **project**
-- **resource_ids**: The resource IDs to which the rule applies. If resource
-  type is `project` then an id of `'*'` applies the rule to all projects.
-- **services**: The list of services to whitelist/blacklist/require.
-
+ 
+- `name`
+  * **Description**: The name of the rule.
+  * **Valid values**: String.
+
+- `mode`
+  * **Description**: The mode of the rule.
+  * **Valid values**: One of `whitelist`, `blacklist` or `required`.
+  * **Note**:  
+     `whitelist` - Allow only the APIs listed in `services`.
```

Comment:
nit: continue the unordered list here with another leading `*`

Reply:
Updated.